software_templates.md

```diff
@@ -0,0 +1,131 @@
+where a template can save a lot of time and overhead. This document will 
+hopefully provide ARFC members with the individual tools to mix and match as 
+they create software projects.
+
+# Existing Templates
+There are a couple open source projects that provide a high quality template 
+that can be useful in software development. If your project falls under the 
+range of capabilities they have you can save a lot of time by using them or 
+adapting them to start your project.
+
+## Shablona
+[Shablona](https://github.com/uwescience/shablona) is a template repository that 
+is designed for scientific python projects. The template comes with everything a 
+small-scale project might need, and can be easily adapted for python projects.
+
+To get a sense of whether or not using the Shablona template is appropriate for 
+your project, checkout the 
+[example](https://github.com/uwescience/shablona/tree/master/examples) provided.
+
+## Cookiecutter
+[Cookiecutter](https://github.com/cookiecutter/cookiecutter) is not a template 
+repository like Shablona, instead it is a command-line utility that
+allows you to build a repository from a "pantry" of templates. This project has 
+an extensive 
+[Read the Docs](https://cookiecutter.readthedocs.io/en/1.7.2/index.html), and 
+it's definitely worth a review before you decide. 
+
+Additional documents that will give you a sense of whether or not to use this 
+utility:
+1. [Getting to Know Cookiecutter](https://cookiecutter.readthedocs.io/en/1.7.2/tutorial1.html) 
+2. [Usage](https://cookiecutter.readthedocs.io/en/1.7.2/usage.html)
+3. [Building a Cookiecutter](https://cookiecutter.readthedocs.io/en/1.7.2/first_steps.html)
+
+# Software Project Parts
+Even if neither of the above templates fit your requirements, there are still 
+some pieces that might benefit your project. JOSS (The Journal of Open Source 
+Software) has 
+a useful [review checklist](https://joss.readthedocs.io/en/latest/review_checklist.html) 
+for reviewers that you should review before beginning your project. 
+
+## Make Files
+Makefile is a program building tool that operates with the make utility to 
+compile or re-compile a program or process through a user-defined command in the 
+command-line. They are extremely useful tools that make it a lot easier for 
+contributors or reviewers to compile and run your project.
+
+#### Resources:
+- [Introduction](https://www.gnu.org/software/make/manual/html_node/Introduction.html)
+- [Writing Makefiles](https://www.gnu.org/software/make/manual/html_node/Makefiles.html#Makefiles)
+- [Tutorial](https://makefiletutorial.com/)
+
+## CMake Files
+CMake generates makefiles, similar Make or a makefile, except that it is much 
+more compatible with C++.
+
+#### Resources:
+- [Overview](https://cmake.org/overview/)
+- [Step-by-step Tutorial](https://cmake.org/cmake/help/latest/guide/tutorial/index.html)
+- [Wiki](https://gitlab.kitware.com/cmake/community/-/wikis/Home)
+
```

Comment:
I would include a section about ``Snakemake`` as well.

Reply:
Thank you for the suggestion, and feedback! If there are other resources or platforms that you use I'd love to include them as well